i18n/ja_JP/LC_MESSAGES/Alltube.po

```diff
@@ -53,11 +53,11 @@ msgstr "よく使われる形式"
 
 #: templates/info.tpl:35
 msgid "Best"
-msgstr "最高品質"
+msgstr "最高品質(mp4形式)"
```

Comment:
MP4 is not mentioned in the source string. (And choosing this option does not always guarantee an MP4 format.)